sync-files/README.md

```diff
@@ -0,0 +1,87 @@
+# sync-files
+
+## Description
+
+This action syncs files between repositories.  
+It uses [peter-evans/create-pull-request](https://github.com/peter-evans/create-pull-request/) for creating pull requests.
+
+Note that you need `workflow` permission for the token if you copy workflow files of GitHub Actions.
+
+## Usage
+
+```yaml
+jobs:
+  sync-files:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate token
+        id: generate-token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.PRIVATE_KEY }}
+
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Run sync-files
+        uses: autowarefoundation/autoware-github-actions/sync-files@add-sync-files
+        with:
+          token: ${{ steps.generate-token.outputs.token }}
+```
+
+Please place `.github/sync-files.yaml` like this.
+
+```yaml
+- repository: autowarefoundation/autoware
+  files:
+    - source: .github/dependabot.yaml
+    - source: .markdownlint.yaml
+    - source: .prettierignore
+    - source: .prettierrc
+```
+
+The specifications are:
+
+- The settings for each repository are listed at the top level.
+
+  ```yaml
+  - repository: org1/repo1
+    files: ...
+
+  - repository: org1/repo2
+    files: ...
+
+  - repository: org2/repo
+    files: ...
+  ```
+
+- The data format for each repository is the following.
+
+| Name                  | Required | Default                                      | Description                                                                              |
+| --------------------- | -------- | -------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| repository            | true     | -                                            | The target repository.                                                                   |
+| ref                   | false    | The default branch of the target repository. | The version of the target repository.                                                    |
+| files/source          | true     | -                                            | The path to the file in the target repository.                                           |
+| files/dest            | false    | The same as `files/source`.                  | The path where to place the synced file in the base repository.                          |
+| files/replace         | false    | `true`                                       | Whether to replace the synced file if it already exists.                                 |
+| files/delete-orphaned | false    | `false`                                      | Whether to delete the synced file if it does not exist in the target repository anymore. |
```

Comment:
I referred https://github.com/BetaHuhn/repo-file-sync-action to decide this format.